packages/backend/src/config/app.ts

```diff
@@ -105,7 +105,12 @@ export const Config = {
   privateKey: parseOrProvisionKey(envString('PRIVATE_KEY_FILE', '')),
 
   graphQLIdempotencyKeyLockMs: envInt('GRAPHQL_IDEMPOTENCY_KEY_LOCK_MS', 2000),
-  graphQLIdempotencyKeyTtlMs: envInt('GRAPHQL_IDEMPOTENCY_KEY_TTL_MS', 86400000)
+  graphQLIdempotencyKeyTtlMs: envInt(
+    'GRAPHQL_IDEMPOTENCY_KEY_TTL_MS',
+    86400000
+  ),
+
+  gracePeriodMs: envInt('GRACE_PERIOD_MS', 86400000)
```

Comment:
This is the grace period for the incoming payments to complete when a payment pointer is deactivated. On deactivation, the `expiresAt` field is updated to now + grace period.

I tried to think of a more specific name but struggled to come up with something that wasn't very verbose. Like `incomingPaymentExpireDelayOnPaymentPointerDeactivation`. Maybe it's best to leave the specificity to the env var documentation (which I have done).

Reply:
Oof yeah, I don't think there is a good & short name for this. I do think verbose is better though, maybe `paymentPointerDeactivationPaymentGracePeriodMs`?

Reply:
I think `paymentPointerDeactivationPaymentGracePeriodMs` is a good descriptive name... I dont mind how long it is personally. I will change to that.